### ISSUE TITLE
Isn't Bloom's Taxonomy a joke?

### ISSUE THREAD

**Issue by rasmusab** [datacamp/example-course-specs#23]
When I used to be at Lund University Cognitive Science we (and everybody else, for that matter) were forced to formulate course criteria using Bloom's Taxonomy. It was a straightjacket everybody hated and it just resulted in a lot of extra work when coming up with learning objectives. And every time you used a taboo verb you got hit on the fingers (you are not allowed to use the word *understand*, for example) 

In summary: Where I used to work Bloom's Taxonomy was considered a joke and I was hoping to never have to see it again 😬 .

I think the learning objectives as they are today, a short list written using the course instructors own words (free use of any verb) is perfect! 😸
